src/main/scala/is/hail/annotations/ReadRowsRDD.scala

```diff
@@ -337,7 +337,6 @@ class ReadRowsRDD(sc: SparkContext,
 
       private var rowSize = in.readLong()
 
-      private val buffer = new Array[Byte](8 * 1024)
```

Comment:
There's another unused buffer in the same file in `RichRDDUnsafeRow`.

Reply:
yup just saw that, updating